articles/libraries/auth0-swift/save-and-refresh-jwt-tokens.md

```diff
@@ -39,32 +39,34 @@ Auth0
 
 ### Credentials Check
 
-It can be useful to perform a quick check for valid credentials as the user can then be directed to authenticate.
+It can be useful to perform a quick sanity check that you have valid credentials, if not the user can then be directed to authenticate.
 
 ```swift
-guard self.credentialsManager.hasValid() else {
-    // Present Login Screen
+guard credentialsManager.hasValid() else {
+    // Present Hosted Login Page
 }
 ```
 
-### Renewing User Credentials
+### Returning User Credentials

```

Comment:
Recovering/Retrieving looks better IMO.

Reply:
ok I thought this originally, will change